packages/rum-core/src/domain/view/viewMetrics/trackScrollMetrics.ts

```diff
@@ -67,3 +88,39 @@ export function computeScrollValues() {
     scrollTop,
   }
 }
+
+export function createScrollValuesObservable(
+  configuration: RumConfiguration,
+  throttleDuration = THROTTLE_SCROLL_DURATION
+): Observable<ScrollValues> {
+  const observable = new Observable<ScrollValues>(() => {
+    function notify() {
+      observable.notify(computeScrollValues())
+    }
+
+    if (window.ResizeObserver) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what about IE11? `ResizeObserver` is not supported there

Reply:
For IE11, the fallback solution is to collect data on the scrolling at regular intervals: 

https://github.com/DataDog/browser-sdk/blob/9ddc001adfbe716f2e38b0922d36a11ac661175e/packages/rum-core/src/domain/view/viewMetrics/trackScrollMetrics.ts#L121-L122

Reply:
oh right, so fallback to setInterval for IE 11 🤔
Do you see any pros / cons to have those two implementations rather than just always using setInterval?

Reply:
cons: 
    - we have two different implementations, which can be troublesome
pro:
    - even if we have two implementations, the two still produce the same values, the only difference lies in the frequency with which these values are produced. For IE we're going to produce new values every second, for other browsers, we're going to produce them only when the page is scrolled or resized
    - on modern browser, since we produce values less often, we have less changes of causing layout shifts/reflows caused by scroll/viewport calculation

Reply:
Indeed, the resize observer approach seems more "polite" than recomputing the values every second, I am wondering if it is not even too agressive for IE 😕
It could worth to double check that it does not degrade performances there.

Did you look into approaches taken by some resize observer polyfills?
In case there is some interesting alternate approach that could be used everywhere

Reply:
Do we have a way to test the SDK on IE? 

I can definitely try an approach with a resize observer polyfill.

Reply:
> Do we have a way to test the SDK on IE?

Yes, with browserstack